frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -1,13 +1,19 @@
+/* eslint-disable */
```

Comment:
We don't want to disable all linting rules. Disable only what you need.

Reply:
updated